index.html

```diff
@@ -17,27 +17,126 @@
   </head>
   <body>
     <div class="container">
-      <header class="header">
-        <div class="logo">ByBook</div>
-        <div class="search-container">
-          <form action="">
-            <input id="search" type="search" placeholder="Search" />
-            <label for="search" class="bx bx-search-alt-2"></label>
-          </form>
-        </div>
+    <header class="header">
+      <div class="logo">ByBook</div>
+      <div class="search-container">
+        <form action="">
+          <input id="search" type="search" placeholder="Search" />
+          <label for="search" class="bx bx-search-alt-2"></label>
+        </form>
+      </div>
 
-        <div class="action-icon">
-          <a href="#" class="login"><i class="bx bxs-user-circle"></i></a>
-          <a href="" class="icon">
-            <i class="bx bx-heart"></i>
-            <span>0</span>
-          </a>
-          <a href="#" class="icon">
-            <i class="bx bx-cart"></i>
-            <span>0</span>
-          </a>
+      <div class="action-icon">
+        <a href="#" class="login"><i class='bx bxs-user-circle'></i></a>
+        <a href="#" class="icon">
+          <i class="bx bx-heart"></i>
+          <span>0</span>
+        </a>
+        <a href="#" class="icon">
+          <i class="bx bx-cart"></i>
+          <span>0</span>
+        </a>
+      </div>
+    </header>
+    <section class="hero-banner-container">
+      <img src="https://dispatch.barnesandnoble.com/content/dam/ccr/homepage/daily/2022/02/01/Gateway_Billboard-C_OMP.jpg" alt="">
+    </section>
+    <section class="book-category">
+      <a href="#"><i class='bx bx-shape-circle' ></i>Fiction
+      </a>
+      <a href="#"><i class='bx bx-knife'></i>Crime
+      </a>
+      <a href="#"><i class='bx bx-heart'></i>Romance
+      </a>
+      <a href="#"><i class='bx bxs-ghost'></i>Horror
+      </a>
+      <!-- <a href="#">Classics -->
+      </a>
+      <a href="#"><i class='bx bx-money-withdraw' ></i>Business
+      </a>
+    </section>
+    <section class="new-release">
+        <div class="new-release-head">
+            <h3>New Release</h3>
+            <button class="btn">
+                <a class="link" href="#">View All</a>
+            </button>
+        </div>
+        <div class="release-container">
+            <div class="ecommerce-card-box">
+                <div class="card-header">
+                    <img src="https://m.media-amazon.com/images/I/71TGnFC-5yL._AC_SY200_.jpg" alt="">
```

Comment:
You could give value to this `alt` attribute and all the subsequent tags, as that helps for accessibility